xtask/src/main.rs

```diff
@@ -0,0 +1,69 @@
+use std::{env, process::Command};
+
+type DynError = Box<dyn std::error::Error>;
```

Comment:
Feel free to refactor this to use anyhow later! It makes error handling very streamlined in small scripts like this one.

Reply:
Have pushed a new PR (https://github.com/google/comprehensive-rust/pull/2707) for this and the other suggestions.